### ISSUE TITLE
Eth & Polygon support

### ISSUE THREAD

**Issue by andrewxhill** [textileio/near-storage-cli#3]
It seems like the quickest path to CLI support for Eth/Polygon would be to change this to `storage-cli` and enable network switching. Curious your thoughts @carsonfarmer

**Comment by carsonfarmer** [comment]
Yup, just a network flag of some kind. It could actually be moved to https://github.com/textileio/storage-js as another sub-package pretty easily. That'd make testing and development a lot easier.
